docs/_includes/ww-admin-tasks-content.md

```diff
@@ -132,17 +142,27 @@ Staff users with the appropriate permissions may create and edit pre-written res
 
 Click on ‘Templates’ in the admin menu. You will see a table of any existing templates. Click on the button below, marked ‘New template’.
 
-Common template responses deal with statuses such as: Report received; Collection scheduled; Container ordered; Report under investigation.
+<img alt="New template button" src="/assets/img/ww-user-guide/WasteWorks new template button.png" />
+
+To create a template:
+1. Input a title. This should be something easy to understand like 'Bin not presented'.
+2. Write more detail in the 'Text' box. This will be displayed on the WasteWorks site to users viewing information on a property's collections.
+3. Also write some text in the 'Text for email alert field'. This text will be emailed to the user if they have opted into alerts.
```

Comment:
This step is optional, if you leave it blank it will use the same text as the Text box (and is slightly quicker).

Reply:
Oh and we don't really use the website for report pages for waste, so this makes less sense due to that, ho hum

Reply:
It's not displayed to on a property, it's displayed on a report, I mean.